workers/unity/Packages/com.improbable.gdk.tools/DownloadCoreSdk.cs

```diff
@@ -25,6 +25,14 @@ internal static class DownloadCoreSdk
 
         private const string DownloadForceMenuItem = "SpatialOS/Download CoreSdk (force)";
 
+        private static readonly List<PluginDirectoryCompatibility> PluginsCompatibilityList = new List<PluginDirectoryCompatibility>
+        {
+            new PluginDirectoryCompatibility("Assets/Plugins/Improbable/Core/OSX", false, new List<BuildTarget> { BuildTarget.StandaloneOSX }, null, true),
```

Comment:
> Copying the lost comment

It would be great if there was a way to link this to the paths in .DownloadCoreSdk.cs. Otherwise if we change the folder hierarchy we have to change hardcoded paths in 2 files.
The problem is that we deliberately wanted to keep DownloadCoreSDK separated from Unity logic so that it can be run separately (in case Unity bugs with menu item). Considering that, I couldn't come up with a lightweight solution to share the paths (one solution that came to my mind is shared config file, but that one seemed a bit too heavy to just avoid duplicating strings in 2 places).
Happy to hear the suggestions!

Reply:
It's in a separate process because for some reason Unity project file generation doesn't include references to the assembly that allows access to the Zipfile functionality, so we can't unzip in a platform-agnostic way. There's no concern with "glitches in the menu or anything - just an annoying Unity quirk.

Regardless, we don't change these paths often, so I don't think it's a huge concern.

Reply:
That makes sense, thanks for the context!